backend/src/test/resources/application-default.yaml

```diff
@@ -11,13 +11,16 @@ spring:
         default_schema: simple_report
 logging:
   level:
-    # NOTE: un-comment any of the below to turn on something interesting
+    # Always have our own debug logging turned on in tests:
+    gov.cdc.usds: DEBUG
     # Hibernate SQL query logging: basically the same as hibernate.show_sql but through slf4j
-    # org.hibernate.SQL: DEBUG
+    org.hibernate.SQL: DEBUG
```

Comment:
Just making sure you want this on by default.

Reply:
Yeah, it was on in the `dev` profile and I didn't port it over when I took that profile out of tests. Having SQL logged in tests is really useful, so as soon as I figured out why it had stopped, I wanted to put it back.